tutorials/parser/src/main/java/com/datatorrent/tutorial/csvparser/Campaign.java

```diff
@@ -0,0 +1,163 @@
+package com.datatorrent.tutorial.csvparser;
+
+import java.util.Date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+
+public class Campaign
+{
+  private int adId;
+  private String adName;
+  private String campaignName;
+  private String parentCampaign;
+  private double campaignBudget;
+  private double bidPrice;
+  private int campaignId;
+  private long securityCode;
+  private boolean weatherTargeting;
+  private boolean active;
+  private boolean optimized;
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd HH:mm:ss")
```

Comment:
We have schema restriction to look for this date format. Please check "schema" in properties file.